src/GameCommand.cpp

```diff
@@ -402,8 +401,7 @@ void GameCommand::LoadOne( const Command& cmd )
 
 	else if( sName == "url" )
 	{
-		m_sUrl = sValue;
-		m_bUrlExits = true;
+		// do nothing
```

Comment:
Changing this to. `// url usage has been deprecated. This block exists for backwards compatibility.`